<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InequalityLoader</class>
 <widget class="QWidget" name="InequalityLoader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InequalityLoader</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_Number">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>#</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label_CaseOut">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_Case">
       <property name="text">
        <string>Case:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Details">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Details</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxShape">
     <item>
      <property name="text">
       <string>star</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>circle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>triangle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>square</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxColor">
     <item>
      <property name="text">
       <string>blue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dark blue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dark green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dark red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>grey</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>black</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonUp">
     <property name="text">
      <string>⋀</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonDown">
     <property name="text">
      <string>⋁</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxInteract">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>⋂</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>⋃</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Remove">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxSkip">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
